src/misc/setup.cpp

```diff
@@ -1135,13 +1135,29 @@ Verbosity Config::GetStartupVerbosity() const
 	return Verbosity::High;
 }
 
+bool CommandLine::IsOption(const std::string &name, cmd_it &it) {
+	if (!(FindEntry(("/" + name).c_str(), it, false)) &&
+	    !(FindEntry(("-" + name).c_str(), it, false)) &&
+	    !(FindEntry(("--" + name).c_str(), it, false)))
+		return false;
+	return true;
+}
+
 bool CommandLine::FindExist(char const * const name,bool remove) {
 	cmd_it it;
 	if (!(FindEntry(name,it,false))) return false;
 	if (remove) cmds.erase(it);
 	return true;
 }
 
+bool CommandLine::FindExistOption(const std::string &name, bool remove) {
+	cmd_it it;
+	if (!IsOption(name, it))
+		return false;
+	if (remove) cmds.erase(it);
```

Comment:
Please use the clang formatter for commits - it will place the action on a separate line from the if statement.

Reply:
`erase(...)` needs a dereferenceable iterator, so if `it` happens to be `end()` or uninitialized, then this bit will crash.   
So this is relying on `IsOption(..)` to assign a valid iterator to `it`.
Suggest asserting `it != it.end()`, just to be sure.

Reply:
Done.